src/genie/libs/parser/slxos/show_version.py

```diff
@@ -0,0 +1,203 @@
+    ---------------------------------------------------------------------------
+    M1      SLX-OS  18r.1.00h                                          STANDBY
+                    18r.1.00h
+    M2      SLX-OS  18r.1.00h                                          ACTIVE*
+                    18r.1.00h
+    L1/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L2/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L3/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L4/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L5/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L6/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L7/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    L8/0    SLX-OS  18r.1.00h                                          ACTIVE
+                    18r.1.00h
+    """
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+        ver_dict = {}
+        result_dict = {}
+
+        # Firmware name:      18r.1.00h
+        p0 = re.compile(r'^Firmware name:\s+(?P<firmware_ver>.+)\s*$')
+        # Build Time:         08:13:43 Apr  7, 2021
+        p1 = re.compile(r'^Build Time:\s+(?P<build_time>.+)\s*$')
+        # Install Time:       07:14:32 Jun  1, 2021
+        p2 = re.compile(r'^Install Time:\s+(?P<install_time>.+)\s*$')
+
+        # System Uptime:   93days 5hrs 32mins 10secs
+        p3 = re.compile(r'^System Uptime:\s+(?P<system_uptime>.*)$')
+
+        # Slot    Name    Primary/Secondary Versions                         Status
+        # ---------------------------------------------------------------------------
+        # M1      SLX-OS  18r.1.00h                                          ACTIVE*
+        #                 18r.1.00h
+        # M2      SLX-OS  18r.1.00h                                          STANDBY
+        #                 18r.1.00h
+        # L1/0    SLX-OS  18r.1.00h                                          ACTIVE
+        #                 18r.1.00h
+        # or
+        # Name     Primary/Secondary Versions
+        # ------------------------------------------
+        # SLX-OS   20.2.2a
+        #          20.2.2a
+        p4 = re.compile(
+            r'^(?P<slot>[A-Z0-9]+[\/[0-9]+]?)?\s*SLX-OS\s+'
+            r'(?P<primary_ver>\d+\w?\.\d+\.\d+\S*)\s*'
+            r'(?P<status>[a-zA-Z]+)?\*?\s*$')
+        p5 = re.compile(r'^\s*(?P<secondary_ver>\d+\w?\.\d+\.\d+[a-z]*)\s*$')
+
+        for line in out.splitlines():
+            line = line.strip()
+
+            m = p0.match(line)
```

Comment:
Please add in comments above each if block with the device output that it corresponds to